src/pages/courses/[slug].tsx

```diff
@@ -285,11 +30,9 @@ export default Course
 export const getServerSideProps: GetServerSideProps = async ({res, params}) => {
   res.setHeader('Cache-Control', 's-maxage=120, stale-while-revalidate')
   const course = params && (await loadCourse(params.slug as string))
-  const dependencies = getDependencies(course.slug)
```

Comment:
there was no compelling reason to do this here since it is "compiled" data anyway so I moved it into the client side render method